### PR TITLE
Document pushgateway admin-api with wipe method

### PR DESCRIPTION
**Pull request by snebel29**
This is complementary to https://github.com/prometheus/pushgateway/pull/285 adding a small update on the admin-api being available now in pushgateway as well

### CODE REVIEW

File: content/docs/operating/security.md [hunk 1]
```diff
@@ -89,6 +89,11 @@ delete the metrics contained within. As the Pushgateway is usually scraped with
 `honor_labels` enabled, this means anyone with access to the Pushgateway can
 create any time series in Prometheus.
 
+The `--web.enable-admin-api` flag controls access to the
+administrative HTTP API which includes functionality such as to wipe all the existing
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: as wiping all

**Comment by snebel29** [comment]
Typo fixed, thanks for pointing it out

File: content/docs/operating/security.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ delete the metrics contained within. As the Pushgateway is usually scraped with
 create any time series in Prometheus.
 
 The `--web.enable-admin-api` flag controls access to the
-administrative HTTP API which includes functionality such as to wipe all the existing
+administrative HTTP API which includes functionality as wiping all the existing
```
Comment (brian-brazil) [inline content/docs/operating/security.md]: You still want the "such"

Comment (snebel29) [reply content/docs/operating/security.md]: ops!

File: content/docs/operating/security.md [hunk 3]
```diff
@@ -89,6 +89,11 @@ delete the metrics contained within. As the Pushgateway is usually scraped with
 `honor_labels` enabled, this means anyone with access to the Pushgateway can
 create any time series in Prometheus.
 
+The `--web.enable-admin-api` flag controls access to the
+administrative HTTP API which includes functionality such as wiping all the existing
```
Comment (beorn7) [inline content/docs/operating/security.md]: Nit: Comma between "API" and "which" (non-restrictive relative clause).

**Pull request merged**